actors/paych/tests/paych_actor_test.rs

```diff
@@ -155,6 +186,83 @@ mod paych_constructor {
             );
         }
     }
+
+    #[test]
+    fn sendr_addr_not_resolvable_to_id_addr() {
+        const TO_ADDR: u64 = 101;
+        let to_addr = Address::new_id(TO_ADDR);
+        let paych_addr = Address::new_id(TEST_PAYCH_ADDR);
+        let caller_addr = Address::new_id(TEST_CALLER_ADDR);
+        #[allow(overflowing_literals)]
```

Comment:
The go test doesn't overflow but reads randomly generated bytes seeded off the input integer.  You don't need to replicate that behavior but you should avoid overflowing here and just choose an arbitrary bls key byte.

Reply:
replaced BLS addr value with non-overflowing value